passes/sat/sim.cc

```diff
@@ -1460,7 +1460,7 @@ struct SimWorker : SimShared
 	{
 		log_assert(top == nullptr);
 		fst = new FstData(sim_filename);
-
+        timescale = fst->getTimescaleString();
```

Comment:
Indentation looks wrong. Yosys uses hard tabs with 8 spaces per tab. Make sure that you have tab settings set correctly in VS Code.

Reply:
This applies to this whole PR